### ISSUE TITLE
Newly created store is of type PKCS12, even when storetype is JKS

### ISSUE THREAD

**Issue by ricoapon** [puppetlabs/puppetlabs-java_ks#406]
## Describe the Bug
Whenever a new store is created, a command is executed to create a new store. If you specify, JKS as storetype, you still get PKCS12 as type.

## Expected Behavior
The store type is JKS.

## Steps to Reproduce
1, Create the following configuration:
```
java_ks { 'some_alias':
        ensure              => latest,
        target              => '/some_path',
        password            => 'some_password',
        certificate_content => '...'
        storetype           => 'jks'
      }
```
2. Run puppet and make sure the store does not exist yet.

## Environment
 - Java_ks version 4.3.1
 - Puppet version 6.27.0
 - Linux RHEL 7

## Additional Context
I found the issue in the code. See `keytool.rb` function `def create`. Based on the storetype different code is executed. The `else` case is done when storetype is `jks`, but no storetype is set in this command. Adding it would solve the issue.

**Comment by pmcmaw** [comment]
Duplicated issue with additional information exists here: https://tickets.puppetlabs.com/browse/MODULES-10568

**Comment by juliantaylor** [comment]
the jira issue says it should be closed for tracking in github, where are you tracking?

**Comment by pmcmaw** [comment]
Hey @juliantaylor I have removed that comment stating it should be tracked in github.

In line with company preferences, we will be tracking in JIRA under a project called CONT (Content&Tooling)
I have moved the JIRA ticket above into this new project and labelled with Customer-Support: https://tickets.puppetlabs.com/browse/CONT-186

If you click on the old issue (https://tickets.puppetlabs.com/browse/MODULES-10568) it will automatically take you to the issue. 

Apologies for the confusion, we are currently trying to bring everything inline.
